pandas/tests/plotting/test_boxplot_method.py

```diff
@@ -194,7 +194,27 @@ def test_color_kwd(self, colors_kwd, expected):
         result = df.boxplot(color=colors_kwd, return_type="dict")
         for k, v in expected.items():
             assert result[k][0].get_color() == v
-
+            
+    @pytest.mark.parametrize(
+        "dark_expected,default_expected",
+        [
+            ({"boxes": "#8dd3c7","whiskers": "#8dd3c7","medians":"#bfbbd9","caps":"#8dd3c7"},
+            {"boxes": "#1f77b4","whiskers": "#1f77b4","medians":"#2ca02c","caps":"#1f77b4"})
+        ],
+    )
+    def test_colors_in_theme(self, dark_expected,default_expected):
+        # GH: 40769
+        df = DataFrame(np.random.rand(10, 2))
+        import matplotlib.pyplot as plt
+        plt.style.use('dark_background')
+        dark_result = df.boxplot(return_type="dict")
+        for k, v in dark_expected.items():
+            assert dark_result[k][0].get_color() == v
+        plt.style.use('defult')
```

Comment:
Should this be 
```suggestion
        plt.style.use('default')
```
?